Update interfaces/payment-handler.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/5ac15fe/whatwg/idl/payment-handler.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/429153951

foolip commented on interfaces/payment-handler.idl:
```diff
@@ -68,7 +75,9 @@ interface PaymentRequestEvent : ExtendableEvent {
   readonly attribute object total;
   readonly attribute FrozenArray<PaymentDetailsModifier> modifiers;
   readonly attribute DOMString instrumentKey;
+  readonly attribute bool requestBillingAddress;
```
It says bool here instead of Boolean, I assume that will cause problems.

autofoolip committed:
Update interfaces/payment-handler.idl

Source: https://github.com/tidoust/reffy-reports/blob/5ac15fe/whatwg/idl/payment-handler.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/429153951

foolip reviewed (approved):
The boolean fix has propagated now. Still need to try running the tests to make sure they aren't broken before merging.

foolip commented:
Filed https://github.com/web-platform-tests/wpt/issues/13024 and then tested locally to confirm test still works.
